gdctools/gdc_dice.py

```diff
@@ -448,17 +484,24 @@ def append_diced_metadata(file_dict, diced_paths, annot, meta_file_writer):
         sample_type = None
         if meta.has_sample(file_dict):
             sample_type = meta.sample_type(file_dict)
+        tcga_barcode = meta.tcga_id(file_dict)
+
 
         # Write row with csv.DictWriter.writerow()
         rowdict.update({
             'case_id'      : meta.case_id(file_dict),
-            'tcga_barcode' : meta.tcga_id(file_dict),
```

Comment:
I recognize that this is old-ish code, but it should nevertheless aim to be agnostic to project, rather than explicitly referencing TCGA